### ISSUE TITLE
Error in executing test script

### ISSUE THREAD

**Issue by fbartolom** [edamov/pushok#81]
I personalised the test script with my app information to get:

```
<?php
//require __DIR__ . '/vendor/autoload.php';

use Pushok\AuthProvider;
use Pushok\Client;
use Pushok\Notification;
use Pushok\Payload;
use Pushok\Payload\Alert;

$options = [
    'key_id' => '255YU6R88K', // The Key ID obtained from Apple developer account
    'team_id' => 'GF9PQ87F68', // The Team ID obtained from Apple developer account
    'app_bundle_id' => 'com.augmentedReality.virtualTags', // The bundle ID for app obtained from Apple developer account
    'private_key_path' => 'AuthKey_255YU6R88K.p8', // Path to private key
    'private_key_secret' => null // Private key secret
];

$authProvider = AuthProvider\Token::create($options);

$alert = Alert::create()->setTitle('Hello!');
$alert = $alert->setBody('First push notification');

$payload = Payload::create()->setAlert($alert);

//set notification sound to default
$payload->setSound('default');

//add custom value to your notification, needs to be customized
$payload->setCustomValue('key', 'value');

$deviceTokens = ['4303AC16C13448D4820345994E898FBD1EE95DD6B702B0133E5C065808FA8BE7'];

$notifications = [];
foreach ($deviceTokens as $deviceToken) {
    $notifications[] = new Notification($payload,$deviceToken);
}

$client = new Client($authProvider, $production = false);
$client->addNotifications($notifications);



$responses = $client->push(); // returns an array of ApnsResponseInterface (one Response per Notification)

foreach ($responses as $response) {
    $response->getApnsId();
    $response->getStatusCode();
    $response->getReasonPhrase();
    $response->getErrorReason();
    $response->getErrorDescription();
}
?>
```
commenting out the first line that linked nowhere, but I get error:

> PHP Fatal error:  Uncaught Error: Class 'Pushok\AuthProvider\Token' not found in /var/www/html/php/push/comodoPush.php:18
> Stack trace:
> #0 {main}
>   thrown in /var/www/html/php/push/comodoPush.php on line 18

**Comment by edamov** [comment]
You need to require `__DIR__ . '/vendor/autoload.php'`

**Comment by fbartolom** [comment]
Fine, I did not notice the new directory: now I have:

> PHP Warning:  Use of undefined constant CURLMOPT_MAX_HOST_CONNECTIONS - assumed 'CURLMOPT_MAX_HOST_CONNECTIONS' (this will throw an Error in a future version of PHP) in /var/www/html/php/push/vendor/edamov/pushok/src/Client.php on line 98
> PHP Warning:  curl_multi_setopt() expects parameter 2 to be int, string given in /var/www/html/php/push/vendor/edamov/pushok/src/Client.php on line 98
> PHP Notice:  Trying to access array offset on value of type null in /var/www/html/php/push/vendor/edamov/pushok/src/Response.php on line 191

**Comment by edamov** [comment]
I didn't test the library on PHP 7.4 yet. Seems those errors appeared in the new php version

**Comment by fbartolom** [comment]
And when are you fixing them? I took so long to get here, and that is a bit disheartening to have got too far...

Dr. Fabrizio Bartolomucci
eMail: fbartolom@gmail.com
iPhone: +39 3474013121

**Comment by edamov** [comment]
Unfortunately I don't have MacBook and Apple developer account for testing pushes with new PHP version soon.
Maybe you can help me with PR?

**Comment by fbartolom** [comment]
I wonder thereafter why you pushed me to install all those pieces of new software, if you did not have any way to test it. What do you mean by PR?

**Comment by edamov** [comment]
It's open source library. I worked on it when I had free time and possibility.
I didn't push you to install it, I just wanted to help you. If you want, you can use any other library on your choice.
PR - pull request. You can investigate the issue and send me changes, and I will merge them into the library.

**Comment by rasmusbe** [comment]
I just ran the unit tests (including those in #89) in the latest 7.4RC and all tests did pass. _BUT_ the unit tests does not test the curl connections.

But I also see that the constant `CURLMOPT_MAX_HOST_CONNECTIONS` should be available since PHP 7.0.7 and cURL 7.30.0 so I guess you are running some old version of php or curl.
